boards/nxp/frdm_mcxn947/doc/index.rst

```diff
@@ -205,6 +205,42 @@ see the following message in the terminal:
    *** Booting Zephyr OS build v3.6.0-479-g91faa20c6741 ***
    Hello World! frdm_mcxn947/mcxn947/cpu0
 
+Flashing to QSPI
+================
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: zephyr/samples/hello_world -DCONFIG_MCUBOOT_SIGNATURE_KEY_FILE=\"bootloader/mcuboot/root-rsa-2048.pem\" -DCONFIG_BOOTLOADER_MCUBOOT=y
+   :board: frdm_mcxn947/mcxn947/cpu0/qspi
+   :goals: flash
+
+
+In order to load Zephyr application from QSPI you should program a bootloader like
+MCUboot bootloader to internal flash. Here are the steps.
+
+.. zephyr-app-commands::
+   :zephyr-app: bootloader/mcuboot/boot/zephyr
+   :board: frdm_mcxn947/mcxn947/cpu0/qspi
```

Comment:
The description declares that it programs the bootloader to the internal (on-chip) flash, but the  board is the "qspi" variant. Could you clarify?

Reply:
@mmahadevan108 Please feel free to correct me here, but my understanding is that the QSPI variant of this board keeps the `boot_partition` within the internal flash region. MCUBoot configures itself to be compiled and programmed to the `boot_partition` within the flash, so the QSPI variant of this board would still have MCUBoot programmed to the internal flash.

Reply:
Thank you, thats correct.